component/webui/src/main/resources/locale/social/Webui_en.properties

```
@@ -962,3 +962,9 @@ space.template.team=Team
 space.template.description.team=A dedicated community where team members can share informations and documents (documents, notes, agenda)
 
 label.inactive=Inactive
+
+#####################################################################################
+#                                   EXternalLabelTag                                #
+#####################################################################################
+external.label.tag=(External)
```

```suggestion
external.label.tag=External
```

Pur the () directly when using the label